presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -48,6 +48,7 @@ services:
       - ${PRESTO_JDBC_DRIVER_JAR}:/docker/volumes/jdbc/driver.jar
       - ../../../conf/tempto/tempto-configuration-for-docker-default.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
       - ../../../target/test-reports:/docker/volumes/test-reports
+      - .presto_history:/root/.presto_history
```

Comment:
This file will be dirty, shouldn't be in the repo.

What about dropping this PR?

Reply:
> What about dropping this PR?

Do not give up so fast. Give me another chance. Will post a different approach soon.